test/units/TEST-75-RESOLVED.sh

```diff
@@ -1084,6 +1084,29 @@ testcase_13_varlink_subscribe_dns_configuration() {
         <(jq -cr --seq  '.configuration[] | select(.ifname == "dns0" and .servers != null and .searchDomains != null) | {"dns0":{servers: [.servers[] | .address], domains: [.searchDomains[] | .name]}}' "$tmpfile")
 }
 
+# Test RefuseRecordTypes
+testcase_14_refuse_record_types() {
```

Comment:
Please add cleanup function something like the following:
```sh
    cleanup() {
        rm -f /run/systemd/resolved.conf.d/refuserecords.conf
        if [[ -e /etc/resolv.conf.bak ]]; then
            rm -f /etc/resolv.conf
            mv /etc/resolv.conf.bak /etc/resolv.conf
        fi
        restart_resolved
    }
    trap cleanup RETURN ERR
```
Otherwise, the setting may be used other test cases.